src/e2e-test/features/aribasource/RunTime.feature

```diff
@@ -0,0 +1,55 @@
+# Copyright © 2022 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+
+@AribaSource
+@Smoke
+@Regression
+Feature: Ariba Source - Run time scenarios
+
+  @BATCH-TS-ARIBA-DSGN-RNTM-01 @BQ_SINK @FILE_PATH @BQ_SINK_CLEANUP @BQ_TEMP_CLEANUP
+  Scenario: Verify user should be able to deploy the pipeline when plugin is configured for System Type prod
```

Comment:
nit - `Verify data is transferred correctly to BQ for System Type prod`

Reply:
Updated